lib/pure/parseutils.nim

```diff
@@ -242,8 +242,9 @@ proc captureBetween*(s: string, first: char, second = '\0', start = 0): string =
   result = ""
   discard s.parseUntil(result, if second == '\0': first else: second, i)
 
-{.push overflowChecks: on.}
-# this must be compiled with overflow checking turned on:
+template integerOutOfRangeError(): ref ValueError =
```

Comment:
Please turn this into a ``proc`` that does the ``raise`` and mark the proc with ``.noinline``, produces better code.